cypress/support/functions.ts

```diff
@@ -148,6 +148,34 @@ Cypress.Commands.add('createApp', ({appName, archiveName, sourceType, customPake
 
   cy.clickEpinioMenu('Applications');
   cy.clickButton('Create');
+
+  // Select the Source Type if needed
+  if (sourceType) {
+    cy.get('.labeled-select').click();
+    cy.contains(sourceType, {timeout: 120000}).click();
+    switch (sourceType) {
+      case 'Container Image':
+        cy.typeValue({label: 'Image', value: archiveName}); 
+        break;
+      case 'Git URL':
+        cy.typeValue({label: 'URL', value: archiveName});
+        cy.typeValue({label: 'Branch', value: 'main'}); 
+        break;
+      case 'Archive':
+        cy.get('.archive input[type="file"]').attachFile({filePath: archiveName, encoding: 'base64', mimeType: 'application/octet-stream'});
```

Comment:
I had to specify `.archive` here because there are two `input[type="file"]` in the page now, because of new` From manifest` button.